Dockerfile

```diff
@@ -17,6 +17,8 @@ ARG TF_ANNOTATION
 ENV TF_ANNOTATION=${TF_ANNOTATION}
 ARG DJANGO_CONFIGURATION
 ENV DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION}
+ARG COVERALLS_REPO_TOKEN
```

Comment:
There should be no changes in Dockerfile (except karma-coveralls string), remove it.